### PR TITLE
GAWB-3141 Test with 20 second retries for Trial Projects

### PR DESCRIPTION
**Pull request by andy7i**
**PR checklist**
- [x] For each library you've modified here, decide whether it requires a major, minor, or no version bump. (Click [here](/broadinstitute/workbench-libs/blob/develop/CONTRIBUTING.md) for guidance)

If you're doing a **major** or **minor** version bump to any libraries:
- [ ] Bump the version in `project/Settings.scala` `createVersion()`
- [ ] Update `CHANGELOG.md` for those libraries
- [ ] I promise I used `@deprecated` instead of deleting code where possible

In all cases:
- [x] Get two thumbsworth of PR review
- [x] Verify all tests go green (CI _and_ coverage tests)
- [x] Squash commits and **merge to develop**
- [x] Delete branch after merge

After merging, _even if you haven't bumped the version_: 
- [ ] Update `README.md` and the `CHANGELOG.md` for any libs you changed with the new dependency string. The git hash is the same short version you see in GitHub (i.e. seven characters). You can commit these changes straight to develop.

### CODE REVIEW

File: GETTING_STARTED.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ export ARTIFACTORY_PASSWORD=getThisFromJenkins
 export ARTIFACTORY_USERNAME=getThisFromJenkinsToo
 ```
 Note: Get the credentials from a Jenkins build job, like rawls-build
+
```
Comment (andy7i) [inline GETTING_STARTED.md:14]: Github UI displays these two lines as: 
"Note: Get the credentials from a Jenkins build job, like rawls-build 3. Publish your snapshot:"

**Review by davidangb — APPROVED** [review]
:+1: change looks fine; I didn't test to see if it is effective.

**Review by rushtong — APPROVED** [review]
👍  If this works, and you have Nicole's thumb, then I approve.

**Comment by ndbolliger** [comment]
👍

**Pull request merged**